src/components/molecules/Card-Product/CardProduct.scss

```diff
@@ -0,0 +1,98 @@
+@import "../../../assets/styles/main.scss";
+$padding-card-common: 0.5rem;
+$description-card-desktop-height: 5rem;
+.product{
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        // grid-template-rows: 2rem 6rem 2rem;
+        padding: 0.5rem;
+        // margin-bottom: 1rem;
+        // font-size: 0.9rem;
+        border-bottom: 1px solid black;
+        justify-items: center;
+        &_title{
+            grid-column: 1 / 3;
+            grid-row: 1 / 2;
+            height: 3.5rem;
+            font-size: $font-medium;
+            @media screen and(max-width:$breakpoint-mobile){
+                height: auto;
+            }
+        }
+        &_card-image{
+            grid-column: 1 / 3;
+            grid-row: 2 ;
+            object-fit: contain ;
+            @media screen and(max-width:$breakpoint-tablet){
```

Comment:
mobile first approach??

Reply:
Tried to convert it to mobile first by giving default styles for mobile.